### ISSUE TITLE
Rename and deploy to PyPI

### ISSUE THREAD

**Issue by cthoyt** [songlab-cal/tape-neurips2019#3]
After reorganizing the code in #1, it would be a good idea to deploy this code to PyPI to even further support reproducibility. However, there is already a package named [tape](https://pypi.org/project/tape/), so it would be necessary to rename first...

**Comment by rmrao** [comment]
We should be able to call the package `tape-benchmark` or `protein-tape` or something. Not sure if we should also rename the imports - it's certainly doable so it's a question of whether something like `from tape import ...` is better or `from tape_benchmark import ...` is better.

**Comment by thomas-a-neil** [comment]
I don't think we need to rename the imports, for example [UMAP](https://umap-learn.readthedocs.io/en/latest/basic_usage.html) doesn't. `pip install umap-learn` and `import umap`
